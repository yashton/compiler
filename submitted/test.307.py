x='5\
654334'