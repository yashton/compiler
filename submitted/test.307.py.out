(ID "x")
(PUNCT "=")
(ERROR "Illegal character ''5\
654334'' on line 1)
