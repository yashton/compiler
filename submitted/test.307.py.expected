(ID "x")
(PUNCT "=")
(LIT "5654334")
(ENDMARKER)
